vuepress/docs/next/docs/create/component-gen-overview.md

```diff
@@ -1,42 +1,44 @@
-  * Enable teams to quickly recognize and adopt code from system integrators and other third parties because the patterns and standards are encoded in the starting point of your applications
+- Begin with running software: By automating the creation of microservices and micro frontends, the user is able to bypass the manual development required to build the default configuration of an Entando project.
+- Customize as desired: The generated code is ready to be developed on and modified to enable features and capabilities.
+- Avoid a learning curve: The Entando Blueprint uses standard tools, technologies, and patterns that can be consumed by developers who are not familiar with the underlying technology.
+
+### Standardize Technology 
+
+- Guarantee technology compliance: The Component Generator encodes business standards for networking, monitoring, build tools, and distribution into the Entando Blueprint to ensure that business requirements are satisfied.
+- Guarantee internal compliance: The Entando Blueprint provides parameters to ensure the code complies with standards and practices of your organization including formatting, testing tools, static analysis tools, and package structures.
+- Jumpstart 3rd party adoption: Teams can quickly recognize and adopt code from system integrators and other third parties because these patterns and standards are encoded into the default application.
 
 ## Generated Component Flow
 
-This section provides an overview of the component generation process. If you want to jump into the code and a tutorial start here: [Entando Component Generator Tutorial](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md)
+The following provides an overview of the component generation process. To view the code and learn how to use the Entando Blueprint, refer to the [Entando Component Generator Tutorial](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md).
 
 1. Create components
-   - The Component Generator will create a Spring Boot microservice and optionally will create React micro frontends for entities that you add to your project
-   - The microservice and generated micro frontends are runnable out of the box and preconfigured to connect to [Entando Identity Management](../consume/identity-management.md)
-   - The code is meant to be edited and enhanced by developers. JHipster provides interactive updates and merges if you need to re-run the generator or add new functionality via JHipster
-   - The microservice includes liquibase and integrated data mocking if you want to generate data in your API calls
+   - The Component Generator creates a Spring Boot microservice and, optionally, React micro frontends for entities that you add to your project.
+   - The microservice and generated micro frontends are out-of-the-box runnable and preconfigured to connect to the [Entando Identity Management System](../consume/identity-management-system.md).
```

Comment:
Isn't the microservice also generated? 

can't help it but never liked the word runnable, though it's totally correct. Here though, runnable out of the box sounds better to me. It also matches the flow of the next set of verb and object
are runnable out of the box
are preconfigured to connect

Reply:
i think there's a subtlety here; the MS is generated for sure but the MFE(s) are optional, so it's almost like this is an abbreviation for "the MFEs that you chose to be generated." i'm fine with (optional) replacing it. 

i hate runnable but like the rework. the flow is improved with "the MFEs are 1 and 2" instead of "the MFEs are 1 and are 2." the second "are" can be removed and the previous order can still exist, but "out of the box" is much stronger as an adjective, leading the subject, than an afterthought. here there is only one object and 2 verbs.

Reply:
disagree on the second point but not at all a big deal